src/views/core/router/router.view.tsx

```diff
@@ -1,11 +1,42 @@
-import { FC } from "react";
+import { ComponentType, FC } from "react";
 import { Navigate, Route, Routes } from "react-router-dom";
 
 import { useEnvContext } from "src/contexts/env.context";
-import routes from "src/routes";
+import { RouteId, routes } from "src/routes";
 import { areSettingsVisible } from "src/utils/feature-toggles";
+import Activity from "src/views/activity/activity.view";
+import BridgeConfirmation from "src/views/bridge-confirmation/bridge-confirmation.view";
+import BridgeDetails from "src/views/bridge-details/bridge-details.view";
+import Home from "src/views/home/home.view";
+import Login from "src/views/login/login.view";
+import NetworkError from "src/views/network-error/network-error.view";
+import Settings from "src/views/settings/settings.view";
 import PrivateRoute from "src/views/shared/private-route/private-route.view";
 
+const components: Record<RouteId, { component: ComponentType }> = {
+  activity: {
+    component: Activity,
+  },
```

Comment:
I think that we can do directly `activity: Activity` the const is called `components`

Reply:
Absolutely